src/mdx/expandVersionedUrl.spec.js

```diff
@@ -23,7 +23,7 @@ describe("expandVersionedUrl", () => {
       ],
 
       [
-        "/Users/monkeypants/camunda-platform-docs/optimize_versioned_docs/version-3.8.0/what-is-optimize.md",
+        "/Users/monkeypants/camunda-platform-docs/optimize_versioned_docs/version-3.9.0/what-is-optimize.md",
```

Comment:
Running tests for this branch uncovered that we never updated the version mapping specs based on our most recent release. That's what these changes are!